### ISSUE TITLE
Grayscale causes text to look terrible

### ISSUE THREAD

**Issue by LoganDark** [bp2008/BetterClearTypeTuner#18]
So... there's a problem with grayscale antialiasing.

With subpixel antialiasing / ClearType turned *on*, this is what some text looks like:

![cleartype](https://user-images.githubusercontent.com/4723091/211145727-a3d40200-44b0-4406-92cc-f26fd7d9d31e.png)

You'd think that turning it to grayscale would look like this:

![grayscale](https://user-images.githubusercontent.com/4723091/211145743-c40ca494-644d-42a6-a71c-c71ae004578e.png)

...but no. what actually happens is:

![fucked](https://user-images.githubusercontent.com/4723091/211145750-1c3b9ffc-4eb0-4063-a095-a4fd15871e51.png)

This is my previous experience turning off ClearType completely. I... hate it. wholly. it's terrible. is there any way for me to turn off the color fringing but without causing all the text to look terrible?

...oh, and this is truly awful:

![image](https://user-images.githubusercontent.com/4723091/211145890-eb51f4cc-826a-4bf3-a456-68afca88ece1.png)

**Comment by bp2008** [comment]
Yes, I also hate how the default version of grayscale antialiasing looks.  Fortunately there is an app called [MacType](https://www.mactype.net/) that lets you customize text rendering a lot further.  Search for the word MacType in this reddit post and they have linked a couple of custom profiles that aren't provided in the standard [MacType](https://www.mactype.net/) download: https://old.reddit.com/r/OLED_Gaming/comments/mbpiwy/lg_oled_gamingpc_monitor_recommended_settings/

**Comment by LoganDark** [comment]
> Yes, I also hate how the default version of grayscale antialiasing looks. Fortunately there is an app called [MacType](https://www.mactype.net/) that lets you customize text rendering a lot further. Search for the word MacType in this reddit post and they have linked a couple of custom profiles that aren't provided in the standard [MacType](https://www.mactype.net/) download: https://old.reddit.com/r/OLED_Gaming/comments/mbpiwy/lg_oled_gamingpc_monitor_recommended_settings/

Thank you so so much!! This is perfect!

![image](https://user-images.githubusercontent.com/4723091/211160489-18be9102-2fee-4bae-b6df-0302b2cd2671.png)

**Comment by LoganDark** [comment]
I eventually had to uninstall MacType [because it's so slow](https://github.com/snowie2000/mactype/issues/914), so I'm now looking for a solution to this problem again :/

**Comment by bp2008** [comment]
Wow, I wonder if this is why task manager is such a comically slow piece of crap on my main system.

**Comment by bp2008** [comment]
Nope.  Task Manager still sucks after stopping and exiting Mactype and also after rebooting.  Just clicking the Processes tab will freeze it for 1-5 seconds.  It is awful.  Mactype doesn't seem to make a difference to performance (though notably, it does affect text rendering where stock Cleartype settings do not affect Task Manager on my system!).

It would be great if Windows did a better job of text rendering, but it is a monumental challenge [just getting them to decide to do anything](https://github.com/microsoft/PowerToys/issues/25595), and I suspect actually improving things would take years due to all the compatibility problems.

**Comment by AMDphreak** [comment]
﻿Windows grayscale isn’t “ClearType with the color removed.” Choosing Grayscale (`FontSmoothingType = 1`) switches to a **separate** stock AA path, and that soft/washed look is just what GDI’s grayscale rasterizer produces. There is no contrast/gamma (or other) knob this app can flip to make that path look like sharp grayscale.

That’s also why MacType can look better here: it **bypasses** Windows’ renderer (hooks / its own rasterization), not because there’s a hidden ClearType setting BetterClearTypeTuner is missing. This tool only exposes what Windows actually honors, so it can’t fix stock grayscale quality.

Closest OS-native options: keep ClearType (and live with fringing), or for DirectWrite/WPF-only apps try a lower `ClearTypeLevel` (blend toward gray). GDI grayscale won’t match either.
